hazelcast/src/test/java/com/hazelcast/map/impl/nearcache/NearCacheTest.java

```diff
@@ -229,6 +229,111 @@ public void testNearCacheEviction_withMapClear() {
         });
     }
 
+    @Test
+    public void testNearCacheStatsWithReadOnlyProcessor() {
```

Comment:
should we add executeOnKey to the method name?

Reply:
Done